src/components/calcite-label/calcite-label.scss

```diff
@@ -5,17 +5,17 @@
 
 // temporary hardcoded values until new font scale and padding is added to calcite-base
 :host([scale="s"]) label {
-  font-size: 10px;
+  font-size: 12px;
```

Comment:
Is it possible to leave these as is in this component and override in the wrapping component? 

Obviously not as clean but, these `calcite-label`s are usually used as wrappers for a `calcite-input` which itself has a larger font size. So in radio-button these labels kind of function as the actual input control itself as far as having the larger size and are probably better off just as a one-off adjustment. 

Apologies for back and forth there - 16px label on top of a 16px input in the usual use case is just too tall whereas in `radio-button` it's the actual control itself.
```
<calcite-label>
  My great label
  <calcite-input />
</calcite-label>
```

Reply:
Cool, just updated `<calcite-label>` to take a custom property for the font-size (with the default fallback) that is now being successfully overridden by `<calcite-radio-button>`.

Reply:
Cool, just updated `<calcite-label>` to take a custom property for the font-size (with the default fallback) that is now being successfully overridden by `<calcite-radio-button>`.